shared/js/initApp.js

```diff
@@ -1,3 +1,6 @@
+// Attempt to save projects every 30 seconds
```

Comment:
Does this apply to both "projects" and "non-projects"? If so, maybe s/`project`/`userSource` or something else that covers the full gamut of cases where used.